modules/40-define-functions/600-type-annotations/description.en.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+name: Type Annotations
+theory: |
+
+  **Type annotations** are an ability to specify parameter types and return values for functions in Python. This is not a mandatory requirement of the language, but it can help programmers in further development, improve the readability of the code and increase its reliability.
```

Comment:
**Type annotations** allow programmers to specify parameter types and return values for functions in Python. It is not a mandatory requirement of the language...